packages/react-components/src/DemocracyLocks.tsx

```diff
@@ -51,7 +51,7 @@ function groupLocks (t: TFunction, bestNumber: BN, locks: Partial<DeriveDemocrac
       .reduce((sorted: Entry[], [{ balance, isDelegated, isFinished = false, referendumId, vote }, blocks]): Entry[] => {
         const isCountdown = blocks.gt(BN_ZERO);
         const header = referendumId && vote
-          ? <div>#{referendumId.toString()} {formatBalance(balance, { forceUnit: '-' })} {vote.conviction.toString()}{isDelegated && '/d'}</div>
+          ? <div>#{referendumId.toString()} {formatBalance(balance, { forceUnit: '-' })} {vote?.conviction.toString()}{isDelegated && '/d'}</div>
```

Comment:
This won't fix the issue - the existence of vote is checked in the preceding line already, so it is defined. However, not sure why a vote won't have conviction? (As per the linked issue)